tutorials/hyperspectral/RasterStack-RGB-Images-in-R-Using-HSI.Rmd

```diff
@@ -397,13 +355,12 @@ the best way to calculate NDVI from hyperspectral data!
 
```

Comment:
We have a graphic that shows NDVI, does it make sense to add here?

Reply:
I bet that it does! What is that graphic file called? I will incorporate in the next iteration of revisions where we parse out NDVI into a separate lesson.